etc/purls.txt

```diff
@@ -0,0 +1,15 @@
+# These package URLs (purls) point to the versions (tags) of external dependencies
+# that are committed to the project. Refer: https://github.com/package-url/purl-spec
+
+# This file is fed to silkbomb to generate the cyclonedx.sbom.json file. Edit this file
+# instead of modifying the SBOM JSON directly. After modifying this file, be sure to
+# re-generate the SBOM JSON file!
+
+# libbson and libmongoc are obtained via cmake/FetchMongoC.cmake.
+pkg:github/mongodb/mongo-c-driver@v1.25.0
+
+# mnmlstc/core is obtained via cmake/FetchMnmlstcCore.cmake.
+pkg:github/mnmlstc/core@v1.1.0
+
+# Catch2 is bundled as src/third_party/catch/include/catch.hpp.
+pkg:github/catchorg/Catch2/@v2.13.7#single_include/catch2/catch.hpp
```

Comment:
Suggest removing Catch2 due to being a test dependency.

From [Centralized Vulnerability Management README](https://docs.google.com/document/d/18DZhrMvdxlFJJS7KaZB1gBNIKispX1W4rlTD-16yGI8/edit):

>  Test- and build-time dependencies (such as the compiler) do not need to be declared.

> We only need to report and fix CVEs in dependencies we actually ship, so reducing any false-positives results in less auditing and updating overhead.